chart/values_nightly.yaml

```diff
@@ -0,0 +1,126 @@
+defaultAirflowRepository: 242864343139.dkr.ecr.ap-northeast-1.amazonaws.com/sendbird/airflow
+defaultAirflowTag: v0.0.1-dev
```

Comment:
Built a new docker image file based on `apache/airflow:1.10.14`, including `Flask-OAuthlib` which is necessary for using Google OAuth login.

Reply:
The image was just for dev purpose. We will create a repo for DAGs and build images there and use it.